### PR TITLE
[RAPTOR-8618] pin dr==2.27.0, release DRUM 1.9.9

### PR DESCRIPTION
**Pull request by yakov-g**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Recently released `datarobot` broke things

### CODE REVIEW

File: custom_model_runner/CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-#### [Current]
+#### [1.9.9] - 2022-09-29
 ##### Changed
 - bump com.datarobot.datarobot-prediction package to 2.2.1
```
Comment (klichukb) [inline custom_model_runner/CHANGELOG.md:9]: add a sentence that it fixed the dependency

Comment (yakov-g) [reply custom_model_runner/CHANGELOG.md:9]: DONE

**Comment by ahjota** [comment]
Out of curiosity, why not pin to `2.28.2`? That's the last in the 2.x series.

**Comment by yakov-g** [comment]
> Out of curiosity, why not pin to `2.28.2`? That's the last in the 2.x series.

Yes, just did

**Review by klichukb — APPROVED** [review]

**Comment by yakov-g** [comment]
> > Out of curiosity, why not pin to `2.28.2`? That's the last in the 2.x series.
> 
> Yes, just did

Eventually, rolling back:
 The conflict is caused by:
[2022-09-29T19:54:14.472Z]     The user requested datarobot==2.28.1
[2022-09-29T19:54:14.472Z]     datarobot-bp-workshop 0.2.3 depends on datarobot==2.27.0

**Review by scottp-ml — APPROVED** [review]

**Review by eric-s-s — APPROVED** [review]

**Pull request merged**